language/ru.json

```diff
@@ -0,0 +1,271 @@
+        ]
+      }
+    },
+    {
+      "label": "Общая обратная связь",
+      "fields": [
+        {
+          "widgets": [
+            {
+              "label": "По умолчанию"
+            }
+          ],
+          "label": "Определить обычную обратную связь для любого интервала баллов",
+          "description": "Нажмите кнопку \"Добавить интервал\" для добавления необходимого количества интервалов. Пример: 0-20% Низкий результат, 21-91% Средний результат, 91-100% Отличный результат!",
+          "entity": "интервал",
+          "field": {
+            "fields": [
+              {
+                "label": "Интервал баллов"
+              },
+              {},
+              {
+                "label": "обратная связь для определённого интервала баллов",
+                "placeholder": "Заполните обратную связь"
+              }
+            ]
+          }
+        }
+      ]
+    },
+    {
+      "label": "Настройки",
+      "description": "Эти настройки помогут определить контроль поведения задания.",
+      "fields": [
+        {
+          "label": "Минимальное число символов",
+          "description": "Установите минимальное число символов, который пользователь должен ввести."
+        },
+        {
+          "label": "Максимальное число символов",
+          "description": "Установите максимальное число символов, который пользователь должен ввести."
+        },
+        {
+          "label": "Размер области ввода",
+          "description": "Размер области ввода в количестве содержащихся строк",
+          "options": [
+            {
+              "label": "1 строка"
+            },
+            {
+              "label": "3 строки"
+            },
+            {
+              "label": "10 строк"
+            }
+          ]
+        },
+        {
+          "label": "Разрешить \"Повторить\"",
+          "description": "При разрешении пользователь может повторить задание."
+        },
+        {
+          "label": "Игнорировать оценивание",
+          "description": "При разрешении пользователь увидит только обратную связь, связвнную с ключевыми словами, но не баллы."
+        },
+        {
+          "label": "Баллы в главной среде",
+          "description": "Используется для начисления баллов в среде только за ответы (скрыто от учащихся)."
+        },
+        {
+          "label": "Проходной процент",
+          "description": "Проценты, необходимые для прохождения"
+        },
+        {
+          "label": "Процент освоения",
+          "description": "Процент, необходимый для освоения. При установлении процента менее 100% снизится соответственно возможный максимум. Он предназначен для того, чтобы дать студентам некоторую свободу действий, а не \"любезно принять\" решения, которые не содержат все ключевые слова."
+        },
+        {
+          "label": "Заглушить чувствительность",
+          "description": "Эта настройка поможет установить, будет ли \"чувствительность\" активирована для всех ключевых слов."
```

Comment:
There's a comma missing at the end of the line, so this is not valid JSON. It would break the editor. I know, I could fix it myself quickly, but I hope this may be useful to check in advance before submitting a pull request. There are plenty of online tools that help you to check JSON objects for validity.

Reply:
Sorry, it's my first experience with codes. Hope that it's fixed now and thank you for new knowledge appropriation support:)

Reply:
@Skiper7 Nothing to feel sorry about!